application-collabora-ui/src/main/resources/Collabora/Code/Translations.xml

```diff
@@ -62,6 +62,13 @@ collabora.attachment.view.title=View using Collabora
 ## Editor
 collabora.editor.onPage=on page
 collabora.editor.error=Error while loading this file. Cause: 
+collabora.editor.close=Close
+collabora.editor.close.hint=Close editor
+collabora.editor.unsaved.close=Keep editing
+collabora.editor.unsaved.info=There are {0}unsaved changes{1}. Are you sure you want to exit without saving?
+collabora.editor.unsaved.autoSave=Note that changes might still be saved by the {0}auto-save{1} feature, if this is enabled.
```

Comment:
I assume you are referring to the auto-save feature provided by Collabora editor, right? So you're saying that this could happen:

* Collabora notifies you that the document has been modified (using ``postMessage``) and you receive the message
* just after that the auto-save feature kicks in and saves the document
* Collabora doesn't notify you that the document has been saved (i.e. it's not modified / dirty any more) -> I have doubts about this
* the user leaves the editor but you think the document is still modified

Reply:
I was actually thinking of another usecase, since I'm sure I will be notified when the auto-save is triggered: 
1. Collabora notifies that the document has been modified (using postMessage) and I receive the message
2. The user clicks on the `close` button and gets a popup about unsaved changes
3. The auto-save is triggered and I get the message, but the popup it's already opened
4. The user clicks on `close without saving` thinking no save was done

Reply:
I see now. Ideally we should put auto save on hold (disable) when the user clicks on close, and resume (re-enable) in case they want to remain in the editing session, but I don't know if Collabora supports this. If not then let's keep the current message.

Reply:
Unfortunately there is not API for this. You can only do it from the server, see https://forum.collaboraonline.com/t/disabling-auto-save-in-collabora-online-code/597/2